.travis.yml

```diff
@@ -70,6 +70,8 @@ jobs:
   - env: EMBER_TRY_SCENARIO=ember-beta BOOTSTRAPVERSION=3
   - env: EMBER_TRY_SCENARIO=fastboot-tests
   - env: EMBER_TRY_SCENARIO=node-tests
+  - env: EMBER_TRY_SCENARIO=ember-default TEST_COMMAND="ember test --launch Firefox"
+  - env: EMBER_TRY_SCENARIO=ember-default BOOTSTRAPVERSION=3 TEST_COMMAND="ember test --launch Firefox"
```

Comment:
Do you think it's worth it to specifically test BS3 in FF? Sure, it *could* capture a bug, but the chances that it happens only for BS3 and FF are rather low I guess. Boils down to a tradeoff between test coverage and CI running time. 

Just thinking aloud here, have no strong opinions...

Reply:
I guess you are right. Tests for IE11 are also run against BS4 only. I guess that's much more likely to catch a bug.